src/python/pants/backend/shell/target_types.py

```diff
@@ -366,6 +366,13 @@ class RunInSandboxSourcesField(MultipleSourcesField):
     expected_num_files = 0
 
 
+class ShellCommandIsInteractiveField(MultipleSourcesField):
```

Comment:
this is needed because previously the presence of `workdir` was used to determine if a request was for an `experimental_run_shell_command`.

That's clearly no longer relevant here.